### ISSUE TITLE
Skinned meshes panic with TAA enabled on 0.14

### ISSUE THREAD

**Issue by komadori** [komadori/bevy_mod_outline#43]
@komadori panics for me with TAA enabled on 0.14 :(

```
thread '<unnamed>' panicked at /home/gena/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-0.20.1/src/backend/wgpu_core.rs:2996:5:
wgpu error: Validation Error

Caused by:
    In a RenderPass
      note: encoder = `<CommandBuffer-(16, 1, Vulkan)>`
    In a draw command, indexed:true indirect:false
      note: render pipeline = `outline_pipeline`
    Incompatible bind group at index 1 in the current render pipeline
      note: Should be compatible an with an explicit bind group layout with label = `skinned_mesh_layout`
      note: Assigned explicit bind group layout with label = `skinned_motion_mesh_layout`
      note: Entry 6 not found in expected bind group layout


note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

_Originally posted by @Shatur in https://github.com/komadori/bevy_mod_outline/issues/27#issuecomment-2251534146_

**Comment by komadori** [comment]
Fixed by 9af01787f8a2f17eb9965e7500c9165d12df15e6. I will make a minor release shortly.

**Comment by Shatur** [comment]
Thanks a lot!
Just tried it out, it no longer crashes. But the outline jitters a bit.

**Comment by Shatur** [comment]
I recorded a video:
https://github.com/user-attachments/assets/66d34de7-6419-4b80-b88f-0cb4eab51308

**Comment by komadori** [comment]
2f371cebddde2a704adfa5fdfaecdba872f33c81 fixes the TAA jitter affecting outlines.

Note that, per https://github.com/komadori/bevy_mod_outline/issues/27#issuecomment-1989721491, outlines are not anti-aliased by TAA because the outlines are drawn after TAA has finished for various reasons.

**Comment by Shatur** [comment]
Thanks, it solved the problem!

**Comment by komadori** [comment]
Released in 0.8.2.
